File: Iniciante/1005 - Média 1/JavaScript/1005.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const lines = input.split("\n");
  const [a, b] = lines.map(Number);
  const WEIGHT_A = 3.5;
  const WEIGHT_B = 7.5;
  const TOTAL_WEIGHT = WEIGHT_A + WEIGHT_B;
  const studentAverage = ((a * WEIGHT_A) + (b * WEIGHT_B)) / TOTAL_WEIGHT;
  console.log(`MEDIA = ${studentAverage.toFixed(5)}`);
}

main();

File: Iniciante/1005 - Média 1/Python/1005.py
def main():
    a = float(input())
    b = float(input())
    WEIGHT_A = 3.5
    WEIGHT_B = 7.5
    TOTAL_WEIGHT = WEIGHT_A + WEIGHT_B
    student_average = ((a * WEIGHT_A) + (b * WEIGHT_B)) / TOTAL_WEIGHT
    print(f"MEDIA = {student_average:.5f}")


if __name__ == "__main__":
    main()
